lib/marketo_api/api/base.rb

```diff
@@ -46,6 +46,38 @@ def version_guard(version)
         end
       end
 
+      # Internal: Converts a hash into a string that can be consumed by
+      # the Marketo API as a query string
+      #
+      # Examples
+      #
+      #   to_query({ source: 'Sales Engage', workspaceName: 'Global' })
+      #   # => 'source=Sales%20Engage&workspaceName=Global'
+      def to_query(options = {})
+        options.to_query.gsub('+', '%20')
```

Comment:
Is it possible to provide feedback to Logan about this? I don't believe this is standard practice for API's and would be better to get aligned on standard practices before the API is heavily used.

Reply:
Syncing with him and Yifu now, will let you know where we land.

Reply:
Got a hold of Yifu, his thoughts are similarly aligned with this StackOverflow thread: https://stackoverflow.com/questions/2678551/when-to-encode-space-to-plus-or-20. This is surpisingly a widely misunderstood, and hotly contested topic. TL;DR + is used for application/x-www-form-urlencoded.

Reply:
Interesting, lots of debate on that StackOverflow thread. Good to know what he is referencing but worth starting a conversation with Steven to discuss design standardization on Marketo's API param usage, also guidelines that we can apply to our own public API. Thanks for following up on that @kuiro5.